tests/machine.test.js

```diff
@@ -1134,4 +1134,177 @@ describe(OlmMachine.name, () => {
             expect(progressListener).toHaveBeenCalledWith(0, 1);
         });
     });
+
+    describe("Request missing secrets", () => {
+        async function getHypotheticalKeyQuery(new_user, new_device, bootstrapCrossSigning) {
```

Comment:
I think it would be good to document what this function does, what its parameters mean, what it returns, etc.

Also doesn't it return a *response*, rather than a *query*? So it probably needs renaming.

Reply:
Updated name and doc. Regarding the naming convention, I am not sure what should be used in this project there is a mix of snake and camel case.
Regarding `bootstrapCrossSigning` I removed it

Reply:
any use of snake case in JS code is a bug, imho

Reply:
resolved as a separate PR